apps/meteor/app/api/server/api.js

```diff
@@ -52,6 +52,25 @@ const getRequestIP = (req) => {
 	return forwardedFor[forwardedFor.length - httpForwardedCount];
 };
 
+const checkPermisisonsForInvocation = (userId, permissionsPayload, requestMethod) => {
```

Comment:
(picky alert)

we might need a better definition on these, but I think we should use more named functions.. 

maybe the definition could be: use arrow functions only when creating local/scoped functions, like a function inside another function. wdyt?

Reply:
yep, I think we're missing a standard.. I'm worried about two things specifically:

- hoist variables and how that affects gc
- better stack traces